playground/css-dynamic-import/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@vitejs/test-css-dynamic-import",
+  "private": true,
+  "version": "0.0.0"
+}
```

Comment:
Without a `package.json`, `cacheDir` will be `playground/node_modules/.vite` and might cause problems in future.